infra/grpc.go

```diff
@@ -29,14 +29,15 @@ import (
 func LogInterceptor() grpc.UnaryServerInterceptor {
 	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
 		start := time.Now()
+		deadline, _ := ctx.Deadline()
```

Comment:
What happens if there is no deadline set? deadline becomes the default time value? Does it make sense to log the absolute deadline instead (or in addition)? (might help with debugging incorrect deadlines higher up the stack)

Reply:
it just returns an empty time.Time so the printed duration will just look weird and be negative:

https://play.golang.org/p/4dMqh1i2EC3